### DISCUSSION TITLE
Custom info pages

### DISCUSSION THREAD

**Discussion by dalf** [searxng/searxng#1330] [Ideas]
Currently there are two info pages. In the future there will be a page for donation, and we can imagine the privacy policy could be an info page too (related to https://github.com/searxng/searxng/pull/1329 ).

In this case we need to provide a easy to customize these pages.
Currently the only is to override the content of the existing .md files.

We could imagine a more easy way for the admin. For example, we can imagine that the `searx/info` directory can be changed to another directory (in a similar way to the static and templates directories). In this case the admins have to synchronize the files between `searx/info` and this new directory, however the implementation is straightforward.

Another way is to:
* let the admins define an additional directory
* search for the .md in the directory first
* fall back to the `/searx/info` after that.
(I'm less confidante about this solution because we have to deal with languages too).

 TLDR: instead of implementation various custom settings ; would it make sense to allow customization of the info pages so the admins can display their own privacy policy & donation pages ?

**Comment by samsapti** [comment]
I think it's a good idea to keep the settings as overrides.

For example, the `privacypolicy_url` could remain for a use case where you want to have a single privacy policy for both SearXNG and other things, such as a personal website.
